pkg/agent/agentserver/tunnel.go

```diff
@@ -127,7 +127,7 @@ func (t *Tunnel) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			klog.Errorf("error sending packet %v", err)
 			continue
 		}
-		klog.Infof("Forwarding on tunnel, packet %v", string(pkt[:n]))
+		klog.Infof("Forwarding on tunnel, packet type: %s", packet.Type)
```

Comment:
Is it intentional to not make this a V(6)?

Reply:
The original goal of the V(6) was to avoid extraneous logging in the apiserver. This isn't part of the apiserver so I was thinking we can have more flexibility/discussion in determining how we want to log. 

If there is a preference for konnectivity server and agent to use V(6) as well, I can make the change.